packages/shared/src/toDisplayString.ts

```diff
@@ -14,7 +14,9 @@ import {
 export const toDisplayString = (val: unknown): string => {
   return val == null
     ? ''
-    : isArray(val) || (isObject(val) && val.toString === objectToString)
+    : isArray(val) ||
+      (isObject(val) &&
+        (val.toString === objectToString || val.toString == null))
```

Comment:
Another thing we could do is check if the toString value is not a function. But I don't know if we want to push toDisplayString too far and I think this was already proposed in the past, either here or in Vue 2 repo

Reply:
oh very good idea, this is implemented in the pr https://github.com/vuejs/vue-next/pull/4337 that was closed